tools/obsolete_unused_reference_files

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+# delay -t must be long enough for testo to complete
+inotifywait -r -m -t 120 reference > log&
+# wait to make sure inotifywait is monitoring
+sleep 10
+./testo
+# wait to make sure inotifywait is done
+sleep 120 
+grep ' OPEN ' log | sed 's/ OPEN //' | sort | uniq > used
+find reference/ -type f | sort | uniq > exists
```

Comment:
Is 'sort -u' (here and above) a suitably universal thing? I think it's POSIX 1003.1.